pkg/k8s/namespaces/crud.go

```diff
@@ -43,11 +47,24 @@ func IsOktetoAllowed(ns *apiv1.Namespace) bool {
 }
 
 // Get returns the namespace object of ns
-func Get(ctx context.Context, ns string, c *kubernetes.Clientset) (*apiv1.Namespace, error) {
-	n, err := c.CoreV1().Namespaces().Get(ctx, ns, metav1.GetOptions{})
+func Get(ctx context.Context, name string, c *kubernetes.Clientset) (*apiv1.Namespace, error) {
+
+	var namespace *apiv1.Namespace
+
+	err := retry.OnError(config.DefaultBackoff, errors.IsTransient, func() error {
```

Comment:
why are we special casing this api call? shouldn't we fail and let the network retry handle this case?

Reply:
I think it's a good idea in general to retry network connections where there's less impact to the overall behavior. Some times the failure is due to a blip in the network, so there's no benefit in having to go all the way back to the top and starting again when a quick second get just fixes it.

 I wanted to start with this one because doing several gets has no side effect.  If we see that this pattern results in a better experience, then we can expand it to other calls. If it makes things more unreliable or complicated, we can go back to the other pattern.